deriv.com.conf

```diff
@@ -0,0 +1,55 @@
+map $http_cf_ipcountry $redirected_url {
```

Comment:
@shudarshon-binary the `/eu` page is not ready yet so we can save this for later 🙏

Reply:
i'll edit the nginx config